README.md

```diff
@@ -143,6 +143,7 @@ formData.append('token', authToken);
 * From that request, copy the "Cookie" header.
 * The values in the field look like `key1=value1; key2=value2; key3=value3;`
 * Get the string `d=XXXXXX;` (where XXX is the secret) and that is your cookie value. It is important to copy the `d=` part and the final `;`.
+* Save the string in its own file (different than the token file).
```

Comment:
You might want to mention instead of "own file" that the option to look for in the man page is --coockiefile.

Also while at it, in the manpage it says `--cookiefile TOKENFILE`, that is wrong and it should say `COOKIEFILE` instead. Could you add it to the pr?